framework/mailer.py

```diff
@@ -73,9 +67,7 @@ def __init__(self):
             # interpolate our variable into them
             template = template.replace(f"{{{{{replace_string}}}}}", replace_value)
 
-        msg = Message(sender=("Sayonika", "noreply@sayonika.moe"), subject=MailSubjects.get(mail_type.value),
+        msg = Message(sender=("Sayonika", "noreply@sayonika.moe"), subject=getattr(MailSubjects, mail_type.value),
                       recipients=[recipient], html=template, charset='utf-8')
-        loop = asyncio.get_running_loop()
 
-        # Send the email in another thread as flask-mail is sync
-        await loop.run_in_executor(self.executor, self.send, msg)
```

Comment:
I mean when I saw the this LoC and the rest of the code for it I knew this wasn't going to work.